genomic_data_service/regulome_atlas.py

```diff
@@ -56,38 +57,73 @@
     100,
 ]
 
-FILE_IC_MATCHED_MAX_PATH_LOCAL = './ml_models/bigwig_files/IC_matched_max.bw'
-FILE_IC_MAX_PATH_LOCAL = './ml_models/bigwig_files/IC_max.bw'
-FILE_IC_MATCHED_MAX_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_matched_max.bw'
-FILE_IC_MAX_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_max.bw'
+FILE_IC_MATCHED_MAX_HG19_PATH_LOCAL = './ml_models/bigwig_files/IC_matched_max.bw'
+FILE_IC_MAX_HG19_PATH_LOCAL = './ml_models/bigwig_files/IC_max.bw'
+FILE_IC_MATCHED_MAX_HG19_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_matched_max.bw'
+FILE_IC_MAX_HG19_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_max.bw'
+FILE_IC_MATCHED_MAX_GRCH38_PATH_LOCAL = './ml_models/bigwig_files/IC_matched_max_GRCh38.bw'
+FILE_IC_MAX_GRCH38_PATH_LOCAL = './ml_models/bigwig_files/IC_max_GRCh38.bw'
+FILE_IC_MATCHED_MAX_GRCH38_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_matched_max_GRCh38.bw'
+FILE_IC_MAX_GRCH38_PATH_REMOTE = 'https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_max_GRCh38.bw'
 
 try:
     TRAINED_REG_MODEL = pickle.load(
         open('./ml_models/rf_model1.0.1.sav', 'rb'))
 except FileNotFoundError:
     TRAINED_REG_MODEL = None
 
-file_IC_matched_max_exists = exists(FILE_IC_MATCHED_MAX_PATH_LOCAL)
-file_IC_max_exists = exists(FILE_IC_MAX_PATH_LOCAL)
+file_IC_matched_max_hg19_exists = exists(FILE_IC_MATCHED_MAX_HG19_PATH_LOCAL)
+file_IC_max_hg19_exists = exists(FILE_IC_MAX_HG19_PATH_LOCAL)
+file_IC_matched_max_grch38_exists = exists(
+    FILE_IC_MATCHED_MAX_GRCH38_PATH_LOCAL)
+file_IC_max_grch38_exists = exists(FILE_IC_MAX_GRCH38_PATH_LOCAL)
```

Comment:
If we wrap the code below in a function, we can probably get rid of these separate existence checks.